### PR TITLE
Completes step 6, works on some feedback

### PR DESCRIPTION
**Pull request by rahulvital**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -110,7 +112,54 @@ describe("GET /api/articles", () => {
...
         .expect(200)
         .then(({ body }) => {
-            expect(body).toBeSortedBy('created_at', { descending: true, coerce: true })
+            const { articles } = body
+            expect(articles).toBeSortedBy('created_at', { descending: true, coerce: true })
+        })
+    })
+})
+
+describe("GET /api/articles/:article_id/comments", () => {
```
Comment (nc-aimee-muter) [inline __tests__/app.test.js:121]: Please add a test to check that we get back an empty array for an existing article with no associated comments

**Review by nc-aimee-muter — COMMENTED** [review]
Nice work, just missing a test and some small improvements suggested to improve already good tests.
